libraries/AP_GPS/AP_GPS_ERB.h

```diff
@@ -66,6 +66,13 @@ class AP_GPS_ERB : public AP_GPS_Backend
         uint8_t fix_type;
         uint8_t fix_status;
         uint8_t satellites;
+        // Introduced in ERB version 0.2.0
+        int8_t numLeapSeconds; ///< leap seconds (0xFF indicates invalid)
+        int32_t baselineN__mm; ///< distance between rover and base - North - in millimeters
```

Comment:
I'm not sure. On one way, it looks ugly and harder to code using it, on the other I do think it allows an easier view of the units. I'm pending more against it though as I don't think the benefit outweighs the disadvantages.